mscore/musescore.cpp

```diff
@@ -2812,6 +2812,23 @@ void MuseScore::showElementContext(Element* el)
       debugger->setElement(el);
       }
 
+//---------------------------------------------------------
+//   reDisplayDockWidget
+//
+//   Helper function to ensure when un-docked widgets are
+//   re-displayed, they are also re-dockable
+//---------------------------------------------------------
+
+void MuseScore::reDisplayDockWidget(QDockWidget* widget, bool visible)
+{
+      if (widget->isFloating() && !widget->isVisible()) {
+            // Ensure the widget is re-dockable if it has been closed and re-opened when un-docked
+            widget->setFloating(false);
+            widget->setFloating(true);
+      }
```

Comment:
Wrong indentation: this brace should be indented one level (6 spaces) more (see the [Handbook](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#Braces)).